dashboard/test/ui/features/star_labs/spritelab/spritelab.feature

```diff
@@ -22,8 +22,9 @@ Scenario: Losing the first level
   And element "#runButton" is visible
   And element "#resetButton" is hidden
 
+@no_mobile
 Scenario: Winning the first level
-  Then I press dropdown number 6
+  Then I click block field "[data-id='make-new-sprite'] > .blocklyEditableText" number 1
```

Comment:
I'm used to these steps being more human-readable. Would it be worth abstracting out the selector to a new step helper so that this line could just say something like:

```
Then I click block make-new-sprite field number 1
```

I realize that this has implications for a bunch of dance-ai tests that would need to be revised, so feel free to acknowledge as out of scope.

Reply:
yeah that is a good follow up task, [filed](https://codedotorg.atlassian.net/browse/CT-319)!